nextroute/solution_format.go

```diff
@@ -130,6 +143,58 @@ func Format(
 	return output
 }
 
+// DefaultStatistics creates default custom statistics for a given solution.
+func DefaultStatistics(solution Solution) CustomStatistics {
+	vehicleCount := 0
+	maxTravelDuration := 0
+	minTravelDuration := math.MaxInt64
+	maxDuration := 0
+	minDuration := math.MaxInt64
+	maxStops := 0
+	minStops := math.MaxInt64
+	for _, vehicle := range solution.Vehicles() {
+		if vehicle.IsEmpty() {
+			continue
+		}
+
+		vehicleCount++
+		duration := vehicle.Duration().Seconds()
+		if int(duration) > maxDuration {
+			maxDuration = int(duration)
+		}
+		if int(duration) < minDuration {
+			minDuration = int(duration)
+		}
+
+		travelDuration := int(vehicle.Last().CumulativeTravelDuration().Seconds())
+		if travelDuration > maxTravelDuration {
+			maxTravelDuration = travelDuration
+		}
+		if travelDuration < minTravelDuration {
+			minTravelDuration = travelDuration
+		}
+
+		stops := vehicle.NumberOfStops()
+		if stops > maxStops {
+			maxStops = stops
+		}
+		if stops < minStops {
+			minStops = stops
+		}
+	}
+
+	return CustomStatistics{
+		UsedVehicles:      vehicleCount,
+		UnplannedStops:    solution.UnPlannedPlanUnits().Size(),
+		MaxTravelDuration: maxTravelDuration,
+		MaxDuration:       maxDuration,
+		MinTravelDuration: minTravelDuration,
+		MinDuration:       minDuration,
+		MaxStopsInRoute:   maxStops,
+		MinStopsInRoute:   minStops,
```

Comment:
The goal of this is to show some default stats so that the experiments come up nicely... I wouldn't want to be completely exhaustive here, for it is just a sample.

Reply:
Yeah, it's hard to draw a line. This is fine. :relaxed: